qa/tasks/cephfs/test_nfs.py

```diff
@@ -104,7 +104,7 @@ def _test_create_cluster(self):
         self._check_nfs_server_status()
         self._nfs_cmd('cluster', 'create', self.export_type, self.cluster_id)
         # Wait for few seconds as ganesha daemon take few seconds to be deployed
-        time.sleep(8)
+        time.sleep(20)
```

Comment:
Is there a way to something like

```python
retry_count = 0
while self._check_nfs_status() != 'running':
    retry_count += 1
    if retry_count > 20:
        self.fail('retry-count exceeded. NFS not running')
    time.sleep(5)
```

There may already be a helper-method somewhere to achieve this

Reply:
@sebastian-philipp Thanks!! I have updated the PR.